Version 1.2.1/os/startup

```diff
@@ -2,16 +2,10 @@ term.clear()
 term.setCursorPos(1, 1)
 print("Cactus OS Arizona")
 print("Loading... ... ... ...")
-term.setCursorPos(1, 3)
-sleep(2)
-textutils.slowPrint("******************")
-sleep(2)
-textutils.slowPrint("Cactus OS, Designed and Made in Arizona!")
-sleep(2)
-textutils.slowPrint("Thank you for using Cactus OS.")
-sleep(2)
-textutils.slowPrint("******************")
-sleep(1)
-textutils.slowPrint("Now starting Cactus OS...")
-sleep(1)
-shell.run(".menu")
\ No newline at end of file
+  term.setCursorPos(1, 3)
+  textutils.slowPrint("******************")
+  textutils.slowPrint("Cactus OS, Designed and Made in Arizona!")
+  textutils.slowPrint("Thank you for using Cactus OS.")
+  textutils.slowPrint("******************")
+  textutils.slowPrint("Now starting Cactus OS...")
+os.run({},".menu")

```

Comment:
Same here as above, if .menu uses any functions of the shell API this will crash.

Reply:
See above